scripts/generate-network-config

```diff
@@ -175,20 +166,21 @@ slaac private
 			lappend dhcpcd "clientid"
 		}
 
-		::fa_config_generator::generate_file "/etc/dhcpcd.conf" $dhcpcd
+		lappend dhcpcd ""
 
 		if {![$config get wireless-network]} {
 			set wlan [::fa_sysinfo::wireless_interface]
-			disable_wpa_supplicant $config $wlan
-		} else {
-			configure_wireless $config
+			lappend dhcpcd "denyinterfaces $wlan"
```

Comment:
It's a bit of a stupid case, but what happens if both wired and wireless are disabled? Not sure if having multiple `denyinterfaces` lines works

Reply:
Multiple `denyinterfaces` seems to work. I went ahead and made it one line to adhere to the dhcpcd.conf man page